src/containers/Treemap.tsx

```diff
@@ -9,27 +9,44 @@ import {
   scaleOrdinal,
 } from 'd3-scale';
 import { schemeCategory10 } from 'd3-scale-chromatic';
+import {
+  propsChanged,
+  stateFromPropUpdates,
+} from 'ihme-ui';
+import findIndex from 'lodash-es/findIndex';
```

Comment:
I'm following a pattern that I don't actually know a lot about. Something David/I have been doing in geospatial.  I think it has to do with tree shaking. For lodash especially, since we only use a few functions at a time, we can drastically reduce the bundle size if we only import the functions we need. Since they are organized in modules with lodash-es it seems preferred to import them with this pattern. Another option is to collect all the lodash-es packages we need into our own lodash package and import them from there, but I don't really know if that would actually benefit us in any real way.